packages/devextreme-scss/scss/widgets/material/chat/_sizes.scss

```diff
@@ -0,0 +1,41 @@
+@use "../sizes" as *;
+
+// adduse
+
+$chat-border-radius: $base-border-radius * 2 !default; // 8px
+$chat-bubble-border-radius: null !default;
+$chat-bubble-padding: null !default;
+$chat-bubble-container-gap: 4px !default;
+$chat-messagebox-padding: null !default;
+$chat-messagebox-textarea-border-radius: null !default;
+$chat-messagebox-buton-gap: null !default;
+$chat-messagebox-gap: 8px !default;
+$chat-messagelist-padding: null !default;
+$chat-messagegroup-gap: 4px !default;
+$chat-messagegroup-padding: null !default;
+$chat-information-gap: 8px !default;
+$chat-avatar-size: 32px !default;
+$chat-messagegroup-alignment-start-template: auto 1fr !default;
+$chat-messagegroup-alignment-start-gap: 12px !default;
+$chat-header-padding: 20px !default;
+$chat-information-font-size: 12px !default;
+
+@if $size == "default" {
+  $chat-messagebox-buton-gap: 10px !default;
+  $chat-bubble-border-radius: 8px !default;
```

Comment:
```suggestion
  $chat-bubble-border-radius: $base-border-radius * 2 !default;
```

Reply:
And you can use this approach out of 'if' blocks

Reply:
[1b7e970409ce772bd7d5fb42ef07ffe1ee0a4c1b]